package.json

```diff
@@ -70,13 +75,17 @@
     "@typescript-eslint/eslint-plugin": "~4.28.3",
     "@typescript-eslint/parser": "~4.28.3",
     "chai": "^4.3.6",
+    "chai-as-promised": "^7.1.1",
     "eslint": "~7.30.0",
     "husky": "~6.0.0",
     "mocha": "^10.0.0",
     "pinst": "~2.1.6",
     "prettier": "2.3.0",
+    "sinon": "^14.0.0",
+    "sinon-chai": "^3.7.0",
     "ts-loader": "~9.2.3",
     "ts-node": "8.6.2",
+    "ts-sinon": "^2.0.2",
```

Comment:
what's the purpose of this lib here?

Reply:
non in specific, I included the whole testing library that I used plus the extra ones that I used on this project, but I could remove it.

Reply:
updated